packages/code-du-travail-data/dataset/fiches_ministere_travail/index.js

```diff
@@ -156,9 +159,12 @@ async function parseFiches(urls) {
       2
     )
   );
-  const fiches = results
-    .map(splitArticle)
-    .reduce((accumulator, documents) => accumulator.concat(documents), []);
+  const fiches = results.map(splitArticle).reduce(
+    (accumulator, documents) =>
+      // we remove section with no title (text before the first sectionTag)
```

Comment:
I'm not sure that I understand this comment

Reply:
old comment :)